File: Service/Billing/RMA.Service.Billing.Contracts/Entities/BadDebtReinstateRequest.cs
using RMA.Service.Admin.MasterDataManager.Contracts.Enums;

using System.Collections.Generic;

namespace RMA.Service.Billing.Contracts.Entities
{
    public class BadDebtReinstateRequest
    {
        public int RoleplayerId { get; set; }
        public List<BadDebtReinstate> BadDebtReinstates { get; set; }
        public string Reason { get; set; }
        public PeriodStatusEnum Period { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/fincare/src/app/billing-manager/models/bad-debt-reinstate-request.ts
import { PeriodStatusEnum } from "projects/shared-models-lib/src/lib/enums/period-status-enum";
import { BadDebtReinstate } from "./bad-debt-reinstate";

export class BadDebtReinstateRequest {
    roleplayerId: number;
    badDebtReinstates: BadDebtReinstate[];
    reason: string;
    period: PeriodStatusEnum;
  }